yarn.lock

```diff
@@ -820,6 +820,15 @@
   resolved "https://registry.yarnpkg.com/@fontsource/inter/-/inter-4.5.15.tgz#eed1873d68755d3b52d6fcfcfa3493118430a512"
```

Comment:
@MeganSteenkamp Did you use `yarn remove` here? I can't see it being removed from the lock file

Reply:
Thanks @tom-blake have updated ❤️